source/assets/javascripts/application.js

```diff
@@ -20,3 +20,9 @@ if (waypointElement !== null) {
 if (/iPad|iPhone|iPod/.test(navigator.userAgent)) {
   document.documentElement.className += " ios-device";
 }
+
+jQuery.get("http://ipinfo.io", function(response) {
+  if (response.country == 'ZA') {
+    $("#London").insertAfter("#CapeTown");
+  }
+}, "jsonp");
```

Comment:
If you're okay with it being directly in the bottom on the contact page - I will do that and move to XMLHttpRequest.